app/models/verification_document.rb

```diff
@@ -5,53 +5,29 @@ def initialize(url:, snap_application:)
   end
 
   def file
-    pdf = Prawn::Document.new
-    magick_object = create_magick_object
-
-    if magick_object.type == "PDF" || magick_object.type == "PBM"
-      magick_object.tempfile
-    else
-      pdf.move_down 30
-      add_header(pdf)
-      rotate_image(magick_object)
+    downloaded_document = RemoteDocument.new(url).download
+    return downloaded_document.tempfile if downloaded_document.pdf?
 
-      pdf.image(magick_object.path, fit: [500, 600])
-      tempfile = Tempfile.new([SecureRandom.hex, ".pdf"])
-      pdf.render_file(tempfile.path)
-      tempfile
-    end
+    tempfile = Tempfile.new([SecureRandom.hex, ".pdf"])
+    pdf = Prawn::Document.new
+    pdf.move_down 30
+    add_header(pdf)
+    pdf.image(downloaded_document.tempfile.path, fit: [500, 600])
+    pdf.render_file(tempfile.path)
+    tempfile
   end
 
   private
 
   attr_reader :url, :snap_application
 
-  def create_magick_object
-    Rails.logger.debug("Creating magick object from #{local_file.path}")
```

Comment:
do we want to keep these logging statements around?

Reply:
I don't know - do we? I don't have an opinion

Reply:
They were put in to try and diagnose an error that... uhh.. proved to not be programmatic in nature. Feel fee to ditch them!